gematria/datasets/bhive_to_exegesis.cc

```diff
@@ -47,7 +47,7 @@ namespace gematria {
 
 BHiveToExegesis::BHiveToExegesis(
     LlvmArchitectureSupport& ArchitectureSupport,
-    llvm::exegesis::LLVMState&& LLVMExegesisState,
+    std::unique_ptr<llvm::exegesis::LLVMState>&& LLVMExegesisState,
```

Comment:
I think passing the `unique_ptr` by value instead of rvalue reference is a bit clearer and more standard. Understood that this is matching existing usage though, so no need to block the PR over this.